### PR TITLE
Update README.md instructions for binary caches

### PR DESCRIPTION
**Pull request by ingenieroariel**
The original instructions caused a full GHC rebuild on OS X.

This mirrors what ./try-reflex does.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,13 +24,14 @@ Obelisk provides an easy way to develop and deploy your [Reflex](https://github.
     1. If you are running NixOS, add this to `/etc/nixos/configuration.nix`:
         ```
         nix.binaryCaches = [ "https://cache.nixos.org/" "https://nixcache.reflex-frp.org" ];
-        nix.binaryCachePublicKeys = [ "ryantrinkle.com-1:JJiAKaRv9mWgpVAz8dwewnZe0AzzEAzPkagE9SP5NWI=" ];
+        nix.binaryCachePublicKeys = [ "cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=" "ryantrinkle.com-1:JJiAKaRv9mWgpVAz8dwewnZe0AzzEAzPkagE9SP5NWI=" ];
```
Comment (alexfmpe) [inline README.md]: I'd expect the nixos key and cache to be on by default?

Comment (ingenieroariel) [reply README.md]: So better to remove both?

Comment (3noch) [reply README.md]: Yeah I think the module system will append these configs together.

Comment (ingenieroariel) [reply README.md]: Removed in a new commit

**Pull request merged**